modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java

```diff
@@ -0,0 +1,827 @@
+  public static final String REPLACE_FLAVOR = "flavor";
+  public static final String REPLACE_CA_NAME = "caName";
+  public static final String REPLACE_RESOLUTION = "resolution";
+
+  public static final String LIVE_STREAMING_URL = "live.streamingUrl";
+  public static final String LIVE_STREAM_NAME = "live.streamName";
+  public static final String LIVE_STREAM_MIME_TYPE = "live.mimeType";
+  public static final String LIVE_STREAM_RESOLUTION = "live.resolution";
+  public static final String LIVE_TARGET_FLAVORS = "live.targetFlavors";
+  public static final String LIVE_DISTRIBUTION_SERVICE = "live.distributionService";
+
+  /** The logger */
+  private static final Logger logger = LoggerFactory.getLogger(LiveScheduleServiceImpl.class);
+
+  private String liveStreamingUrl;
+  private String streamName;
+  private String streamMimeType;
+  private String[] streamResolution;
+  private MediaPackageElementFlavor[] liveFlavors;
+  private String distributionServiceType = DEFAULT_LIVE_DISTRIBUTION_SERVICE;
+  private String serverUrl;
+  private Cache<String, Version> snapshotVersionCache = CacheBuilder.newBuilder().expireAfterWrite(5, TimeUnit.MINUTES).build();
+
+  /** Services */
+  private DownloadDistributionService downloadDistributionService; // to distribute episode and series catalogs
+  private SearchService searchService; // to publish/retract live media package
+  private SeriesService seriesService; // to get series metadata
+  private DublinCoreCatalogService dublinCoreService; // to setialize dc catalogs
+  private CaptureAgentStateService captureAgentService; // to get agent capabilities
+  private ServiceRegistry serviceRegistry; // to create publish/retract jobs
+  private Workspace workspace; // to save dc catalogs before distributing
+  private AssetManager assetManager; // to get current media package
+  private AuthorizationService authService;
+  private OrganizationDirectoryService organizationService;
+
+  private long jobPollingInterval = JobBarrier.DEFAULT_POLLING_INTERVAL;
+
+  /**
+   * OSGi callback on component activation.
+   *
+   * @param context
+   *          the component context
+   */
+  protected void activate(ComponentContext context) {
+    BundleContext bundleContext = context.getBundleContext();
+
+    serverUrl = StringUtils.trimToNull(bundleContext.getProperty(SERVER_URL_PROPERTY));
+    if (serverUrl == null)
+      logger.warn("Server url was not set in '{}'", SERVER_URL_PROPERTY);
+    else
+      logger.info("Server url is {}", serverUrl);
+
+    @SuppressWarnings("rawtypes")
+    Dictionary properties = context.getProperties();
+    if (!StringUtils.isBlank((String) properties.get(LIVE_STREAMING_URL))) {
+      liveStreamingUrl = StringUtils.trimToEmpty((String) properties.get(LIVE_STREAMING_URL));
+      logger.info("Live streaming server url is {}", liveStreamingUrl);
+    } else {
+      logger.warn("Live streaming url was not set in '{}'. This is a problem when not using MHPearl capture agents!",
```

Comment:
Please remove reference reference to MHPearl here - perhaps something like

+      logger.info("Live streaming url not set in '{}'. Streaming urls must be provided by capture agent properties.")

(suggesting info rather than warn because it's not necessarily a config error).